HOWTO_DEVELOP.rst

```diff
@@ -1,21 +1,26 @@
-In order to develop plonetheme.barceloneta you need to execute on the root of
-the package::
+In order to develop plonetheme.barceloneta, make sure you use a ``node`` version >= 18
+and execute on the root of the package::
 
-     npm install
+     yarn install
```

Comment:
Do we have known minimum or maximum versions of `yarn`?

Reply:
After discussion with johannes I've reverted back to "pure" npm.